tests/DocumentationExamplesTest.php

```diff
@@ -1411,29 +1411,38 @@ function testCausalConsistency()
 
         // Prep
         $client = new Client($this->getUri());
-        $test = $client->selectDatabase(
+        $items = $client->selectDatabase(
             'test',
             [ 'writeConcern' => new WriteConcern(WriteConcern::MAJORITY) ]
-        );
-        $items = $client->test->items;
+        )->items;
 
         $items->drop();
         $items->insertOne(
             [ 'sku' => '111', 'name' => 'Peanuts', 'start' => new UTCDateTime() ]
         );
 
         // Start Causal Consistency Example 1
+        $items = $client->selectDatabase(
```

Comment:
I know we have just done this in ``:1414``, but it now needs to be part of the example, and we needed ``$items`` in 1414 so we can make the ``drop``/``insertOne`` to make the test execute.